### ISSUE TITLE
Lack of error message on date filters

### ISSUE THREAD

**Issue by joelvdavies** [ral-facilities/datagateway#908]
**Description**:
To comply with WCAG success criterion 3.3.1, the date filters in dataview should display an error message when an invalid date is supplied in the same way as is done on the search and download pages. The contrast of this label should also be checked with #907.

**Acceptance criteria**:
- [ ] Date filters display an error message when an invalid date is entered

**Comment by joelvdavies** [comment]
Turns out this was actually a bug with the date pickers not re-rendering. Fixed in PR #910.
